app/utils/views.js

```diff
@@ -205,20 +226,27 @@ module.exports = {
   AWAITING_CAPTURE_REQUEST: {
     view: 'errors/charge_confirm_state_completed',
     locals: {status: 'successful'},
-    analyticsPage: '/success_return'
+    analyticsPage: '/success_return',
+    terminalView: true
   },
 
-  display: function (res, viewName, options) {
-    let action = lodash.result(this, viewName)
+  display: function (req, res, viewName, options) {
     options = options || {}
+    let action = lodash.result(this, viewName)
     options.viewName = viewName
     if (!action) {
       logger.error('VIEW ' + viewName + ' NOT FOUND')
       options = {viewName: 'error'}
       action = this.ERROR
     }
+    const redirectToServiceImmediatelyOnTerminalState =
+      res.locals &&
+      res.locals.service &&
+      res.locals.service.redirectToServiceImmediatelyOnTerminalState
```

Comment:
this can be `_.get(res, 'locals.service. redirectToServiceImmediatelyOnTerminalState', false)`